webapp/components/admin_console/storage_settings.jsx

```diff
@@ -202,6 +204,25 @@ export default class StorageSettings extends AdminSettings {
                     disabled={this.state.driverName !== DRIVER_S3}
                 />
                 <BooleanSetting
+                    id='amazonS3SignV2'
+                    label={
+                        <FormattedMessage
+                            id='admin.image.amazonS3SignV2Title'
```

Comment:
I think it's fine to leave it as a true/false value and change the working slightly to `Enable Legacy Signature V2 for S3`.